fenix/app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -240,6 +240,7 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             fromHomeFragment,
         ).also {
             inlineAutocompleteEditText = it.view.findViewById(R.id.mozac_browser_toolbar_edit_url_view)
+            inlineAutocompleteEditText.increaseTapArea(TAP_INCREASE_DPS)
```

Comment:
`TAP_INCREASE_DPS` has a value of 8dp while for `increaseTapArea` to help in this case we only need 4dp.